src/main/java/frc/lib/refvalues/FloatRefValue.java

```diff
@@ -0,0 +1,17 @@
+package frc.lib.refvalues;
+
+public class FloatRefValue {
+    public float value;
```

Comment:
same thing, why public if you have a get and set methods?

Reply:
Same answer